samples/apps/copilot-chat-app/webapi/Program.cs

```diff
@@ -96,6 +103,41 @@ private static void AddServices(IServiceCollection services, ConfigurationManage
         services.AddSemanticKernelServices(configuration);
     }
 
+    private static void AddAuth(this IServiceCollection services, ConfigurationManager configuration)
+    {
+        var useHttp = configuration.GetSection("UseHttp").Get<bool>();
+        var authMethod = configuration.GetSection("Auth:Type").Get<string>();
+
+        if (useHttp && authMethod?.ToUpperInvariant() != "NONE")
+        {
+            throw new ArgumentException("Using HTTP (as opposed to HTTPS) is only supported when the auth type is set to None");
```

Comment:
Could you add a second parameter to the exception to mark the offending property (i.e. "UseHttp")?